portal-frontend/src/app/features/notifications/edit-submission/edit-submission.component.ts

```diff
@@ -129,8 +136,18 @@ export class EditSubmissionComponent implements OnDestroy, AfterViewInit {
         }
         break;
       case EditNotificationSteps.Transferees:
+        //DO NOTHING
```

Comment:
Do I get it right that this will be addressed in next tickets?

Reply:
Nope, that is correct. All transferee actions are saved via the dialog and the page itself has no transient state to save.